public/data/object/examples.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "0d08a9ff-5683-4d3d-a7b8-dce74f37cd3b",
+    "name": "Produit vide",
```

Comment:
Maybe we should use "objet" here and not "produit" (and in the selector title too)?

Reply:
Frankly I *hate* the term "object" and would apprcoate avoiding it as much as possible in the UI, but that's just… me, so…